src/execution_plan.c

```diff
@@ -16,10 +16,10 @@
 #include <event2/event.h>
 #include "lock_handler.h"
 
-#define INIT_TIMER  struct timespec _ts,_te;
+#define INIT_TIMER  struct timespec _ts = {0}, _te = {0};
 #define GETTIME(t)  clock_gettime(CLOCK_REALTIME, t);
-#define START_TIMER GETTIME(&_ts);
-#define STOP_TIMER  GETTIME(&_te);
+#define START_TIMER if(GearsConfig_GetProfileExecutions()) GETTIME(&_ts);
+#define STOP_TIMER  if(GearsConfig_GetProfileExecutions()) GETTIME(&_te);
 #define DURATION    ((long long)1000000000 * (_te.tv_sec - _ts.tv_sec) \
                     + (_te.tv_nsec - _ts.tv_nsec))
 #define ADD_DURATION(d) STOP_TIMER; \
```

Comment:
shouldn't this also have the if(GearsConfig_GetProfileExecutions())

Reply:
Do you mean ADD_DURATION? I guess so.

Reply:
Resolved via 331f9aa